.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+| Question                                  | Answer
+| ---------------------------------------- | ------------------
+| **JIRA issue**                          | EZP-XXXXX
+| **Type**                                   | feature/bug/improvement
+| **Target eZ Platform version** | `v3.x` - please update `x` accordingly
+| **BC breaks**                          | yes/no
+| **Doc needed**                       | yes/no
+
+<!-- Replace this comment with Pull Request description -->
+
+#### Checklist:
+- [ ] Provided PR description.
+- [ ] Tested the solution manually.
+- [ ] Provided automated test coverage.
+- [ ] Checked that target branch is set correctly (master for features, the oldest supported for bugs).
```

Comment:
This should be probably rephrased as this repo won't really handle any bugfixes.

Reply:
Dropped parenthesis